docs/samples/Microsoft.ML.Samples/Dynamic/ImageAnalytics/ConvertToGrayScale.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.IO;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ConvertToGrayscaleExample
+    {
+        public static void ConvertToGrayscale()
+        {
+            var mlContext = new MLContext();
+
+            // Downloading a few images, and an images.tsv file, that contains a list of the files, from the dotnet/machinelearning/test/data/images/
+            // if you inspect the fileSystem, after running this line, the 
```

Comment:
>the [](start = 71, length = 3)

- the sentence looks like its not complete.
- I think some commas can be removed (Downloading a few images and an images.tsv, which contains a list of the files from the dotnet/machinelearning... )